requirements/base.txt

```diff
@@ -93,5 +93,5 @@ redis==3.3.11
 pandas
 wiki
 django-bulk-update
-elasticsearch
```

Comment:
@iamonuwa you need to be a lil careful while adding in code to ensure you don't accidentally delete something else